src/drivers/serac.cpp

```diff
@@ -51,7 +51,6 @@ void defineInputFileSchema(axom::inlet::Inlet& inlet, int rank)
   // Verify input file
   if (!inlet.verify()) {
     SLIC_ERROR_ROOT(rank, "Input file failed to verify.");
```

Comment:
You confirmed this calls exit on all ranks?

Reply:
Yep, when the calling (root) process terminates [all the others will as well](https://www.open-mpi.org/doc/v4.0/man1/mpirun.1.php#sect16).